src/renderer/components/configure-recording-set-page/show-detail.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="170" height="66.258" viewBox="0 0 170 66.258"><defs><style>.a,.c{fill:none;stroke:#707070;}.a{font-size:30px;font-family:SourceHanSansCN-Regular, Source Han Sans CN;}.b{opacity:0.9;}</style></defs><g transform="translate(-910.5 -992.742)"><text class="a" transform="translate(995.5 1054)"><tspan x="-83.085" y="0">Show Detail</tspan></text><g class="b" transform="translate(1018 993.242) rotate(90)"><path class="c" d="M39.219,280.424,17.971,259.177a1.734,1.734,0,0,1,2.452-2.452l19.543,19.531,19.543-19.531a1.734,1.734,0,0,1,2.452,2.452L40.714,280.424a1.054,1.054,0,0,1-1.5,0Z" transform="translate(280.736 -17.466) rotate(90)"/></g></g></svg>
```

Comment:
Can "Show Detail" be extracted out? I don't think it belongs in the icon

Reply:
Zhou - I downloaded the 'Show Detail' svg directly off the mockup. So if you would like it to be different, can you tell me the best way to edit svgs or give me an svg that is what you are looking for?

Reply:
SVG is also XML-based, so it looks very similar to HTML if you format it. I removed `<text>` element and `.a` from the `<style>` element.
```
<svg xmlns="http://www.w3.org/2000/svg" width="170" height="66.258" viewBox="0 0 170 66.258">
    <defs>
        <style>.b{fill:none;stroke:#707070;}.a{opacity:0.9;}
        </style>
    </defs>
    <g transform="translate(-910.5 -992.742)">
        <g class="a" transform="translate(1018 993.242) rotate(90)">
            <path class="b"
                  d="M39.219,280.424,17.971,259.177a1.734,1.734,0,0,1,2.452-2.452l19.543,19.531,19.543-19.531a1.734,1.734,0,0,1,2.452,2.452L40.714,280.424a1.054,1.054,0,0,1-1.5,0Z"
                  transform="translate(280.736 -17.466) rotate(90)"/>
        </g>
    </g>
</svg>
```